docs/source/benchmarking.rst

```diff
@@ -55,8 +72,10 @@ Aggregate across simulated communities (when dataset index is unique)
     ])
     df.to_csv("benchmarks.tsv", sep='\t', index=True, header=True)
 
-Aggregate across simulated communities (when dataset index is `not` unique)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+When dataset index is `not` unique
```

Comment:
This seems less descriptive to me than the previous header. Each of these header sections will be indexed along the side-bar in readthedocs. In this case, I think more descriptive section headers would be easier to quickly look-up

Reply:
See comment above